src/pages/our-leadership.mdx

```diff
@@ -0,0 +1,72 @@
+# oSTEM Executive Committee and Board of Directors
+
+
+![Image of Lilian Martinez](images/lilian-martinez.jpg "Lilian Martinez")
```

Comment:
Can we better organize these so we don't end up with a million things in the images directory?

Maybe `images/headshots/lilian-martinez.jpg` or `images/volunteers/lilian-martinez.jpg`?